integrations/atlassian/confluence/client.go

```diff
@@ -75,3 +76,33 @@ func connStatus(i *integration) sdkintegrations.OptFn {
 		}
 	})
 }
+
+// connTest is an optional connection test provided by the integration
+// to AutoKitteh. It is used to verify that the connection is working
+// as expected. The possible results are "OK" and "error".
+func connTest(i *integration) sdkintegrations.OptFn {
+	return sdkintegrations.WithConnectionTest(func(ctx context.Context, cid sdktypes.ConnectionID) (sdktypes.Status, error) {
+		if !cid.IsValid() {
+			return sdktypes.NewStatus(sdktypes.StatusCodeWarning, "init required"), nil
+		}
+
+		vs, err := i.vars.Get(ctx, sdktypes.NewVarScopeID(cid))
+		if err != nil {
+			return sdktypes.InvalidStatus, err
+		}
+
+		baseURL, err := apiBaseURL()
+		if err != nil {
+			return sdktypes.NewStatus(sdktypes.StatusCodeError, "invalid Atlassian base URL"), nil
+		}
+
+		token := vs.GetValueByString("oauth_AccessToken")
+
+		res, err := accessibleResources(nil, baseURL, token)
+		print(res)
+		if err != nil {
+			return sdktypes.NewStatus(sdktypes.StatusCodeError, "failed to connect to Confluence instance"), nil
```

Comment:
Unlike `connStatus`, I think it's more useful in `connTest` to return `err.Message()` rather than an opaque message. Same in all other integrations.

Reply:
Done